dist/highlight.css

```diff
@@ -0,0 +1,3 @@
+.knotx-devtool-highlight {
+  background-color: lightblue;
```

Comment:
what about case, when a highlight element still has a class which add a background colour parameter to them? 
!important is not a good practice but in this case it could be a justifiable reason.

WDYT? 

@xolir ?

Reply:
Makes sense to me